scripts-vagrant_provision/local.yml

```diff
@@ -0,0 +1,12 @@
+---
+nodejs_version: "16.20.2"
+npm_version: "9.6.4"
+
+python_version: "3.10.3"
+
+docker_version: "19.03.1"
```

Comment:
Can we use 19.03.5? See https://github.com/timescale/timescaledb/issues/5414

It's an error that Cristian got as well